tools/js-mgmt-sdk-release-tools/ci.yml

```diff
@@ -34,6 +34,7 @@ variables:
   NodeVersion: '14.x'
   VAR_ARTIFACT_NAME: "drop"
   VAR_BUILD_ARTIFACT_STAGING_DIRECTORY: $(Build.ArtifactStagingDirectory)
+  NugetSecurityAnalysisWarningLevel: 'none'
```

Comment:
You should import the globals.yml file https://github.com/Azure/azure-sdk-tools/blob/main/eng/pipelines/templates/variables/globals.yml

Reply:
Note you might have to change the format of your variables in order to both import a variable template and define variables. See https://github.com/Azure/azure-sdk-for-net/blob/main/eng/pipelines/mgmt.yml#L169.

Reply:
fixed.